### ISSUE TITLE
Query log: low contrast in block button

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#81]
> The contrast between the small text and background is not sufficient. The contrast ratio is 4.43:1 when it should be 4.5:1 for text smaller than 18px.

<img width="76" height="40" alt="Image" src="https://github.com/user-attachments/assets/8a5f3a6b-2f0a-4cac-9dd6-bbd851f4d65b" />

[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum)
